### PR TITLE
Blocked well add interpretation and feature convenience

### PR DESCRIPTION
**Pull request by emmanesbit**
Added convenience function to quickly create wellbore interpretation and wellbore feature objects for a blocked well object. 
Other minor bug fixes.

### CODE REVIEW

File: resqpy/well.py [hunk 1]
```diff
@@ -865,7 +866,7 @@ def load_from_xml(self, node, hdf5_source_model = None):
       md_datum_part = relatives_model.part_for_uuid(md_datum_uuid)
       assert md_datum_part, 'md datum part not found in model'
       self.md_datum = MdDatum(self.model, md_datum_root = relatives_model.root_for_part(md_datum_part))
-      ds_uuid = bu.uuid_from_string(rqet.find_tag(rqet.find_tag(node, 'DeviationSurvey'), 'UUID'))
+      ds_uuid = bu.uuid_from_string(rqet.find_tag_text(rqet.find_tag(node, 'DeviationSurvey'), 'UUID'))
```
Comment (connortann) [inline resqpy/well.py:869]: I saw Andy wrote some functions for nested tag lookup, could perhaps be useful?

```suggestion
      ds_uuid = bu.uuid_from_string(rqet.find_nested_tags_text(node, ['DeviationSurvey', 'UUID']))
```

File: resqpy/well.py [hunk 2]
```diff
@@ -3007,6 +3012,12 @@ def xyz_marker(self, active_only = True):
       xyz = self.trajectory.xyz_for_md(md)
       return xyz, rqet.uuid_for_part_root(self.trajectory.crs_root)
 
+   def create_feature_and_interpretation(self):
```
Comment (connortann) [inline resqpy/well.py:3015]: Could have a docstring if it's a public method (or else, could give an underscore prefix to the name to make this a private method):

```suggestion
   def create_feature_and_interpretation(self):
      """Instantiate new empty WellboreFeature and WellboreInterpretation objects
      
      Uses the Blocked well citation title as the well name
      """
```

**Review by connortann — APPROVED** [review]
Added two comments for suggested tweaks, up to you whether to change. Otherwise looks good!

**Comment by emmanesbit** [comment]
> Added two comments for suggested tweaks, up to you whether to change. Otherwise looks good!

I think I've accepted them both, good spot on the nested_tags_text and missing docstring, thanks!

**Pull request merged**